### ISSUE TITLE
Context for childViewHolder

### ISSUE THREAD

**Issue by GLObus303** [bignerdranch/expandable-recycler-view#301]
Hey,

I am dynamically creating textviews in each childItem, whatsoever it uses probably uses same context and it's the main fragment view. I dont fully understand the whole context meaning, but would it be possible to get separeted context for each childView holder?

My problem is that textViews are throughout expanding and collapsing of each item just stacking up. I could just delete existing textViews everytime I create new item. But I dont find it a good practice to recreate it all the time over again.

**Comment by paul-turner** [comment]
@GLObus303 Sorry for taking a while to answer your question.

For the first question: In android, any view that you have showing up for your fragment or activity will use the same root context object from your activity. There is no need to get a separated context, your views are tied to that activity and should be deleted when the activity is destroyed. This article may help get you some more background about context: https://possiblemobile.com/2013/06/context/

The second one is a little hard to follow unfortunately. If you are setting up the recycler view correctly there should be no fear of needing to recreate/delete the views every time. Instead you'll be recycling the other text views and simply changing the text. If you have a code sample I might be able to provide more direction, but it sounds like you may want more experience with a basic RecyclerView before diving into the expandable part here.

**Comment by GLObus303** [comment]
Hey,

fine by me. I can imagine my question was a result of some big confusion from the old ME. 

Thanks for the patience
